project.py

```diff
@@ -114,6 +134,79 @@ def create_trackbars(self):
         cv.createTrackbar('lo', 'result', self.lo, 255, self.trackbar_callback)
         cv.createTrackbar('hi', 'result', self.hi, 255, self.trackbar_callback)
 
+    def check_change(self, image, mask):
+        imageOld = cv.imread("slide"+repr(self.numberOfSlides)+".png",
+                                cv.CV_LOAD_IMAGE_COLOR)
+        maskOld = cv.imread("mask.png", cv.CV_LOAD_IMAGE_GRAYSCALE)
+        if(maskOld is None):
+            return True
+
+        if(mask.shape != maskOld.shape):
+            return False
+
+        applyMask = cv.bitwise_and(maskOld, mask)
+        imageOld = cv.bitwise_and(imageOld, imageOld, mask=applyMask)
+        checkImage = cv.bitwise_and(image, image, mask=applyMask)
+
+        orb = cv.ORB()
+        kp1, ds1 = orb.detectAndCompute(imageOld, None)
+        kp2, ds2 = orb.detectAndCompute(checkImage, None)
+        bf = cv.BFMatcher(cv.NORM_HAMMING, crossCheck=True)
+        matches = bf.match(ds1, ds2)
+
+        val = float(len(matches))/len(kp1)
+        if(val > self.similarity):
+            print "Is the same"
+
+        return False
+
+    def get_sorted_rectangle(self, cnt, image):
```

Comment:
@Foxiy I personally do not know what to expect to get from a function called `get_sorted_rectange`. From short, sweet and to the point it seems to be missing the last part.